doc/modules/mod_jingle_sip.md

```diff
@@ -1,22 +1,22 @@
 ### Module Description
 
-This module enabled Jingle to SIP and SIP to Jingle translator.
-When this module is enalbed, MongooseIM will intercept any iq set jingle stanza with action:
+This module enables Jingle to SIP and SIP to Jingle translation.
+When this module is enabled, MongooseIM will intercept any Jingle IQ set stanza with action:
 * session-initiate
 * session-terminate
 * session-accept
 * transport-info
 
-and translate it to SIP messages with approprite SDP content based on the details in the Jingle stanza.
+and translate it to SIP messages with appropriate SDP content based on the details in the Jingle stanza.
```

Comment:
What details?

Reply:
That's complicated :) The type of SIP message is based on the action attr in the jingle element. SDP content is based on the media and content description deeper in the stanza. Do we want to cover that in the doc?

Reply:
Could be a dedicated page but I think should be definitely covered, and not in a tutorial IMO. Raw reference is even more important than tests description.